tilequeue/wof.py

```diff
@@ -66,33 +75,194 @@ def fetch_wof_neighbourhoods(url):
         except ValueError:
             continue
 
-        x, y = pyproj.transform(latlng_proj, merc_proj, lng, lat)
+        file_hash = row[hash_idx]
+
+        label_x, label_y = pyproj.transform(latlng_proj, merc_proj, lng, lat)
+        label_position = shapely.geometry.Point(label_x, label_y)
 
-        neighbourhood = Neighbourhood(wof_id, name, x, y)
-        yield neighbourhood
+        neighbourhood_meta = NeighbourhoodMeta(
+            wof_id, name, label_position, file_hash)
+        yield neighbourhood_meta
 
 
-class WofNeighbourhoodFetcher(object):
+def fetch_wof_url_meta_neighbourhoods(url):
+    r = requests.get(url, stream=True)
```

Comment:
Fair enough - it's a one-off process anyway, so fingers crossed we can do it without hitting any 503s.